gnowledge/gstudio
# Announce of Unit

Run `fab update_data` and try announcing a Unit (base_unit).

_What to test:_
1. Create a base_unit.
2. Add grade, subject, description and modules to it.
3. Add activities, build lessons
4. Add Asset and add tag `raw@material` to few of them.
5. Announce it. Test with uploading an image which will eventually be the banner pic.

- After Announcement, check the above performed updates reflect in new unit.
- This will be `announced_unit`.
- All the data, except notebook will come and sit in this new `announced unit`.
- Assets having the said `tag` ONLY will be pulled into this unit. 
- This `announced unit` will also be added in the respective module's `collection_set`. However, the module detail will show only `base_unit` objects for now.

Note: `announced_unit` will be treated same as `base_unit`, i.e 
1. An `announced_unit` can also be announced.
2. An `announced_unit's` lesson structure can also be modified 
3. All actions that are possible in `base_unit` can be performed in `announced_unit` also.

Other updates:
Modified `fabfile.py` to run `ATs.csv` file after `filldb`.
